omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Install omnibus
-gem 'omnibus', git: 'https://github.com/DataDog/omnibus-ruby.git', branch: 'datadog-5.5.0'
+gem 'omnibus', git: 'https://github.com/DataDog/omnibus-ruby.git', branch: 'greg/patch'
```

Comment:
let's undo this

Reply:
I'll merge the other PR and then remove it